packages/components-vue/package.json

```diff
@@ -33,32 +33,32 @@
   },
   "dependencies": {
     "@porsche-design-system/components-vue": "3.0.0-alpha.2",
-    "vue": "^3.2.45",
+    "vue": "^3.2.47",
```

Comment:
I noticed that we have: `"vue": "^2.7.10",` in `/packages/storefront/package.json` and `"vue": ">=3.0.0 <4.0.0"` in `/packages/components-vue/projects/vue-wrapper/package.json` - should we also update it there?

Reply:
There is a difference between the vue package which contains the vue 3 wrappers and our storefront, which is still on vue 2. See dependencies .md file.